src/System.Net.Http/src/System/Net/Http/Headers/MediaTypeHeaderValue.cs

```diff
@@ -4,6 +4,7 @@
 
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Text;
```

Comment:
Cannot we revert the StringBuilderCache change now to avoid such Mono specific changes?

Reply:
They just made a PR which will bring this exact same change to their master.